src/shared/services/commandInterpreterHelper.js

```diff
@@ -195,5 +195,6 @@ const interpret = (cmd) => {
 }
 
 export default {
-  interpret
+  interpret,
+  commands: availableCommands.map((command) => command.name)
```

Comment:
This is unfortunately not correct. The name is just for identification for internal use and `http`, `play-remote` amongst others aren't commands. 
We could add a `commands: ['get', 'post']`, `commands: ['play']` etc. for those who are commands. 
Would that help?